Nexus.Sdk.Token.Tests/Helpers/MockTokenServerProvider.cs

```diff
@@ -22,12 +22,12 @@ public Task<SignableResponse> ConnectAccountToTokensAsync(string accountCode, IE
             throw new NotImplementedException();
         }
 
-        public Task<AccountResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null)
+        public Task<AccountResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, string? customerIPAddress = null, string? customName = null)
         {
             throw new NotImplementedException();
         }
 
-        public Task<SignableResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, IEnumerable<string> tokenCodes, string? customerIPAddress = null)
+        public Task<SignableResponse> CreateAccountOnAlgorandAsync(string customerCode, string publicKey, IEnumerable<string> tokenCodes, string? customerIPAddress = null, string? customName = null)
```

Comment:
Should we also add it to the Stellar ones?

Reply:
Good idea, I have added the stellar ones :)